packages/lib/src/components/internal/Countdown/useCountdownA11yReporter.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef } from 'preact/hooks';
+import { CountdownA11yReporter } from './CountdownA11yReporter';
+import useSRPanelContext from '../../../core/Errors/useSRPanelContext';
+import useCoreContext from '../../../core/Context/useCoreContext';
+import { CountdownTime } from './types';
+
+export const useCountdownA11yReporter = (time: CountdownTime): void => {
+    const { i18n } = useCoreContext();
+    const { srPanel } = useSRPanelContext();
+    const reporter = useRef<CountdownA11yReporter>(null);
```

Comment:
`useRef` keeps reinitiating new instance if we set `initial value` to `new CountdownA11yReporter({ i18n, srPanel })`. 
Hence I have to use `reporter.current = new CountdownA11yReporter({ i18n, srPanel });` in the `useEffect`.